src/base/static/js/utils.js

```diff
@@ -1,35 +1,4 @@
 var self = (module.exports = {
```

Comment:
Note that all the changes in the `utils.js` file are to refactor Underscore and jQuery usage. In some cases I refactored use of these libraries, in other cases I just removed code that is no longer used.